Samples/security_ipsec/interfaces/.terraform/modules/modules.json

```diff
@@ -0,0 +1 @@
+{"Modules":[{"Key":"","Source":"","Dir":"."}]}
```

Comment:
Can you please remove the .terraform, .tfstate, .tfstatebackup files. This is local to your environment.